initialization/initialization.go

```diff
@@ -345,7 +345,7 @@ func (init *Initializer) initFile(computeProviderID uint, fileIndex int, numLabe
 }
 
 func (init *Initializer) updateSessionNumLabelsWritten(numLabelsWritten uint64) {
-	init.numLabelsWritten.Add(int64(numLabelsWritten))
+	init.numLabelsWritten.Store(int64(numLabelsWritten))
```

Comment:
`Uint64` was added in go 1.19: https://pkg.go.dev/sync/atomic#Uint64.

I suggest changing the type of `init.numLabelsWritten` to `atomic.Uin64` and simplifying accesses to:
```suggestion
	init.numLabelsWritten.Store(numLabelsWritten)
```